src/Helpers/AsBackgroundJobs.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Give\Helpers;
+
+use ActionScheduler;
+
+/**
+ * @see https://actionscheduler.org/
+ *
+ * @unreleased
+ */
+class AsBackgroundJobs
+{
+    /**
+     * @unreleased
+     *
+     * @param string $hook     The hook to trigger.
+     * @param array  $args     Arguments to pass when the hook triggers.
+     * @param string $group    The group to assign this job to.
+     * @param bool   $unique   Whether the action should be unique.
+     * @param int    $priority Lower values take precedence over higher values. Defaults to 10, with acceptable values falling in the range 0-255.
+     *
+     * @return int The action ID. Zero if there was an error scheduling the action.
+     */
+    public static function enqueueAsyncAction(
```

Comment:
Just looking at how we're using this, it might make more sense to create a facade, then all the methods will be static by default but can still be mockable

Reply:
Resolved in 0efdd386c483557d1cd5f41ce61b2e565158947f